src/iniReader/inireader.cpp

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Strictly speaking, this library belongs to third_party/.

Reply:
Done